src/components/Calculator.js

```diff
@@ -0,0 +1,26 @@
+const Calculator = () => (
+  <div className="container">
+    <div className="result">0</div>
+    <div className="key">AC</div>
+    <div className="key">+/-</div>
+    <div className="key">%</div>
+    <div className="operator">/</div>
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Calculator.js%20file.-,In%20Calculator.js%2C%20create%20a%20React%20component%20that%20matches%20this%20design%3A,-Implement%20the%20design), Please replace the division operator with this symbol `÷` instead to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/220390944-9fedbb89-feb1-4ce0-bdca-57e958ae9ada.png) |![image](https://user-images.githubusercontent.com/68796232/220390924-d51b87ee-7490-4d13-85d5-0c485794c6f3.png) |

Reply:
Noted with thanks @Omar-Muhamad.